
Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-bd66t[0m
{"level":"info","ts":1593178220.9265182,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593178220.927195,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-m72l7[0m
{"level":"info","ts":1593178221.8124793,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1593178221.8188257,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-ablazleon-jxpy-pr-4-pr-build-6w7zz-nzdbc[0m
{"level":"info","ts":1593178231.1063108,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593178232.4845016,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*********/**py.git @ master in path /workspace/source"}

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59,4:06c1cd9cecd91800334c856b2cefcca5758c661a
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow 06c1cd9cecd91800334c856b2cefcca5758c661a: 6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59:
DEBUG: ran git fetch --unshallow origin 06c1cd9cecd91800334c856b2cefcca5758c661a: 6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59
DEBUG: ran git reset --hard 6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59 in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge 06c1cd9cecd91800334c856b2cefcca5758c661a
DEBUG: ran git merge 06c1cd9cecd91800334c856b2cefcca5758c661a in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git log --format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e 6cf8aa971b6f96f4b04bb0cdcbe3a52a08095b59..HEAD
Merged SHA 06c1cd9cecd91800334c856b2cefcca5758c661a with commit message 'Update app.py' into base branch master

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
E0626 13:30:37.460844      10 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
[36mINFO[0m[0001] Using dockerignore file: /workspace/source/.dockerignore 
[36mINFO[0m[0001] Retrieving image manifest python:onbuild     
E0626 13:30:37.918368      10 metadata.go:154] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
E0626 13:30:37.921434      10 metadata.go:166] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
[36mINFO[0m[0002] Retrieving image manifest python:onbuild     
[36mINFO[0m[0003] Built cross stage deps: map[]                
[36mINFO[0m[0003] Retrieving image manifest python:onbuild     
[36mINFO[0m[0004] Retrieving image manifest python:onbuild     
[36mINFO[0m[0005] Executing 3 build triggers                   
[36mINFO[0m[0005] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:dc2a839bd6d525fd9783a6852939774c55dc14024a49397cd18ef840aae363e1... 
[36mINFO[0m[0006] Using caching version of cmd: COPY requirements.txt /usr/src/app/ 
[36mINFO[0m[0006] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:8bf2abf0a84228eb1cc03999f3ee3ce7361230c16e28eae7af74285f6291b7d4... 
[36mINFO[0m[0007] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0007] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:ed65078882af0c8e895c363bd43be2d0ce3220519fd2c96c27c6cd1967a0d73f... 
[36mINFO[0m[0007] No cached layer found for cmd COPY . /usr/src/app 
[36mINFO[0m[0007] cmd: EXPOSE                                  
[36mINFO[0m[0007] Adding exposed port: 8080/tcp                
[36mINFO[0m[0007] Unpacking rootfs as cmd COPY . /usr/src/app requires it. 
[36mINFO[0m[0024] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0024] Found cached layer, extracting to filesystem 
[36mINFO[0m[0024] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0024] Found cached layer, extracting to filesystem 
[36mINFO[0m[0024] COPY . /usr/src/app                          
[36mINFO[0m[0024] Resolving 141 paths                          
[36mINFO[0m[0024] Taking snapshot of files...                  
[36mINFO[0m[0024] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:ed65078882af0c8e895c363bd43be2d0ce3220519fd2c96c27c6cd1967a0d73f to cache now 
[36mINFO[0m[0024] COPY requirements.txt .                      
[36mINFO[0m[0024] Resolving 1 paths                            
[36mINFO[0m[0024] Taking snapshot of files...                  
[36mINFO[0m[0024] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:6ef23eac7d9ebbf6e2417d0412cbed1971b231b2d437ea864afaad97fcc38f23 to cache now 
[36mINFO[0m[0024] ENV PORT 8080                                
[36mINFO[0m[0024] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0024] EXPOSE 8080                                  
[36mINFO[0m[0024] cmd: EXPOSE                                  
[36mINFO[0m[0024] Adding exposed port: 8080/tcp                
[36mINFO[0m[0024] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0024] ENTRYPOINT ["python"]                        
[36mINFO[0m[0024] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0024] CMD ["app.py"]                               
[36mINFO[0m[0024] No files changed in this command, skipping snapshotting. 

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-postbuild-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-make-preview[0m
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-4-1/" Chart.yaml
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-4-1/" ../*/Chart.yaml
sed -i -e "s|repository:.*|repository: gcr.io\/envjenkinsk8udacitycapstone\/**py|" values.yaml
sed -i -e "s/tag:.*/tag: 0.0.0-SNAPSHOT-PR-4-1/" values.yaml
echo "  version: 0.0.0-SNAPSHOT-PR-4-1" >> requirements.yaml
** step helm build
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: storage.googleapis.com https://storage.googleapis.com/chartmuseum.jenkins-x.io
Successfully added Helm repository storage.googleapis.com.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: chartmuseum.jenkins-x.io http://chartmuseum.jenkins-x.io
Successfully added Helm repository chartmuseum.jenkins-x.io.

Showing logs for build [32mablazleon-jxpy-pr-4-pr-build-6w7zz-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-jx-preview[0m
Creating a preview
Found commit author match for: ********* with email address: adrian.blazquez.leon@alumnos.upm.es
Created environment *********-**py-pr-4
Namespace **-*********-**py-pr-4 created 
expose:
  Annotations:
    helm.sh/hook: post-install,post-upgrade
    helm.sh/hook-delete-policy: hook-succeeded
  config:
    domain: 34.91.102.68.nip.io
    exposer: Ingress
    http: "true"
preview:
  image:
    repository: gcr.io/envjenkinsk8udacitycapstone/**py
    tag: 0.0.0-SNAPSHOT-PR-4-1

Updating PipelineActivities *********-**py-pr-4-1 which has status Running
Preview application is now available at: http://**py.**-*********-**py-pr-4.34.91.102.68.nip.io
